certification/artifacts/artifacts.go

```diff
@@ -20,59 +25,40 @@ const DefaultArtifactsDir = "artifacts"
 
 func init() {
 	// set the singleton to the default value.
-	ads = DefaultArtifactsDir
+	cwd, _ := os.Getwd()
+	ads = filepath.Join(cwd, DefaultArtifactsDir)
```

Comment:
Could we call `Reset()` here?

Reply:
I had that same thought. I did it in a previous iteration, but changed it. But coming back to it, I think I will.